storage/statedb/cache_fastcache.go

```diff
@@ -0,0 +1,32 @@
+package statedb
+
+import "github.com/VictoriaMetrics/fastcache"
+
+type LocalCache struct {
```

Comment:
The name `LocalCache` suggests that there is a `RemoteCache` struct, but there isn't. 
Also, `LocalCache` sounds like an interface, not a struct. 
How about just wrapping `fastcache.cache` as `statedb.FastCache`?

Reply:
1. RemoteCache will be introduced later soon.
2. When `fastcache.cache` is wrapped to `statedb.FastCache`, type assertion is needed in every method of `statedb.FastCache` to use methods of `fastcache.cache`. In that case, newly introduced interfaces should returns an error also.

Reply:
I am gonna try to find a new name replacing `LocalCache` later. If you have any idea, please let me know.

Reply:
What I meant for "wrapping fastcache.cache as statedb.FastCache" is to change the name of current `LocalCache` to `FastCache`, not changing the current interfaces. Let's proceed as what you intended and check if additional change is needed later. I think it is good for now.

Reply:
OK. I will consider your comment in the next PR.